### PR TITLE
One page article

### PR DESCRIPTION
**Pull request by Amalcxc**
i am having an issue with my git its not pushing. and I do not have time the deadlines near.

### CODE REVIEW

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [x]  Add the "Editor's picks" section using the <aside> tag


### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
...
+
+flexbox
+
+Live Demo
+
+https://amalcxc.github.io/New-York-Times/
+
+Show your support
+
+Give a ⭐️ if you like this project!
```
Comment (christianotieno) [inline README.md:28]: ## The `README` can further get improved to look more presentable.
The `README` has some useful information on what the project is all about. In my view, I think we can make it look more professional to have people understand in-depth what the project is all about in one place. With that, I would suggest we use this [template](https://github.com/microverseinc/readme-template) as an inspiration in adding some crucial information like a screenshot of the page, the author's (you in this case) social links, how to well clone and run the project locally, and have an actual live link this [one](https://raw.githack.com/Amalcxc/New-York-Times/one-page-article/newyork.html) of the actual live page. Right now, this is where users are getting redirected. 👇 

readme

That is more or like the live version of the `README` itself.  I have a suggestion that you can use to generate a live version of the page using this [tool](https://raw.githack.com/). You are welcome to use [this](https://thecompetentdev.com/weeklyjstips/tips/7_serve_files_from_github/) tutorial on how you can go about this as well. What do you think? (note: this is just but one of the ways you can achieve this. You don't need to follow this solution, thank you.)
##
### Here is a summary of things you might want to focus on while working on these suggestions so as to have a `README` that is more descriptive as indicated in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/).
- [x] Have a live link to the page.
- [x] Have a screenshot of the page.
- [ ] Have your social links in the author's section of the `README`.
- [ ] Have instructions on how to well clone and run the project locally.
- [ ] You might want to check on grammatical issues, they are bound to appear from time to time when working with documentations.

File: newyork.css [hunk 2]
```diff
@@ -5,14 +5,21 @@ body{
 	
 }
```
Comment (christianotieno) [inline newyork.css:7]: ## Name change of file suggestion.
- [ ] What do you think about my proposal to rename this file to `index.css`?

File: newyork.html [hunk 3]
```diff
@@ -20,12 +20,23 @@
         <label for="toggle">&#9776;</label>
```
Comment (christianotieno) [inline newyork.html:20]: ## Name change of file suggestion.
- [ ] What do you think about my proposal to rename this file to `index.html`?

File: newyork.html [hunk 4]
```diff
@@ -20,12 +20,23 @@
...
...-1.5 1.3v.1l1.5 1.3V16c-.8.5-1.7.6-2.5.6-1.5 0-2.8-.4-3.9-1.6l4.1-2V6l-5 2.2c.5-1.3 1.6-2.2 2.6-2.9l-.1-.2c-3 .8-5.7 3.5-5.7 6.9 0 4 3.3 7 7 7 4 0 6.6-3.2 6.6-6.5h-.2c-.6 1.3-1.5 2.5-2.6 3.1v-4.1l1.6-1.3v-.1L140 8.8v-3c1.5 0 3-1 3-2.9zm-8.7 11l-1.2.6c-.7-.9-1.1-2.1-1.1-3.8 0-.7.1-1.5.3-2.1l2.1-.9-.1 6.2zm12.2-12h-.1l-2 1.7v.1l1.7 1.9h.2l2-1.7v-.1l-1.8-1.9zm3 14.8l-.8.5-1-1V9.3l1-.7-.2-.2-.7.6-1.8-2.1-2.9 2 .2.3.7-.5.9 1.1v6.5l-1.3 1 .1.2.7-.5 2.2 2 3-2-.1-.3zm16.7-.1l-.7.5-1.1-1V9.3l1-.8-.2-.2-.8.7-2.3-2.1-3 2.1-2.3-2.1L154 9l-1.8-2.1-2.9 2 .1.3.7-.5 1 1.1v6.5l-.8.8 2.3 1.9 2.2-2-.9-.9V9.3l.9-.6 1.5 1.4v6l-.8.8 2.3 1.9 2.2-2-.9-.9V9.3l.8-.5 1.6 1.4v6l-.7.7 2.3 2.1 3.1-2.1v-.3zm8.7-1.5l-2.5 1.9-2.5-2v-1.2l4.7-3.2v-.1l-2.4-3.6-5.2 2.8v6.8l3.5 2.5 4.5-3.6-.1-.3zm-5-1.7V8.5l.2-.1 2.2 3.5-2.4 1.5zm14.1-.9l-1.9-1.5c1.3-1.1 1.8-2.6 1.8-3.6v-.6h-.2c-.2.5-.6 1-1.4 1-.8 0-1.3-.4-1.8-1L176 9.3v3.6l1.7 1.3c-1.7 1.5-2 2.5-2 3.3 0 1 .5 1.7 1.3 2l.1-.2c-.2-.2-.4-.3-.4-.8 0-.3.4-.8 1.2-.8 1 0 1.6.7 1.9 1l4.3-2.6v-3.6h-.1zm-1.1-3c-.7 1.2-2.2 2.4-3.1 3l-1.1-.9V8.1c.4 1 1.5 1.8 2.6 1.8.7 0 1.1-.1 1.6-.4zm-1.7 8c-.5-1.1-1.7-1.9-2.9-1.9-.3 0-1.1 0-1.9.5.5-.8 1.8-2.2 3.5-3.2l1.2 1 .1 3.6z"></path></svg>
```
Comment (christianotieno) [inline newyork.html:30]: ## Turning the page's logo to a link
- [ ] I think turning this logo into a link would be appropriate as it is an actual link on the [original](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html) page, don't you think so?

File: newyork.html [hunk 5]
```diff
@@ -64,27 +75,27 @@ <h1>Space Ripples Reveal Big Bang’s Smoking Gun</h1>
 
         <div >
           <div >
-            <img class="image1" alt src="18cosmos-superJumbo.jpg">
-            <p>Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.Credit...Rick Friedman for The New York Times</p>
+            <img class="image1" alt src="images/18cosmos-superJumbo.jpg">
+            <p class="main-article">Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.Credit...Rick Friedman for The New York Times</p>
             
           </div>
```
Comment (christianotieno) [inline newyork.html:82]: ## Improvement on Page's navbar & main image styling as seen on large screens. 
- [x] The original page has its elements in the right, middle and left bordered evenly to give a sense of enough space on the far end of the page's `navbar`. In our case, we have the icons on the left and the buttons on the right less of space as expected on the borders of the page. Kindly increase the border space of these elements. There need to be space between these elements and the page's edge.
- [x] The page's main image seems to be bigger than expected. Would you mind reducing its size?
- [ ] The main article's heading is styled in such a way that the workings are wrapping on the next line at the `Bang's` word and the words are indented too. The social icons beneath it are also lighter in colour. There is also the horizontal line that should be much thinner than what we currently have and much shorter as well. Would you mind making these changes so as the clone page resemble the original page in this aspect?

- Below are the comparisons you can use as a guide.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/Amalcxc/New-York-Times/one-page-article/newyork.html)|
|:--:|:--:|
|main-image-header|main-image-header-clone|

File: newyork.html [hunk 6]
```diff
@@ -98,66 +109,137 @@ <h6>March 17, 2014</h6>
...
+               <p class="main-article">Inflation has been the workhorse of cosmology for 35 years, though many, including Dr. Guth, wondered whether it could ever be proved.</p>
 
-               <p>If corroborated, Dr. Kovac’s work will stand as a landmark in science comparable to the recent discovery of dark energy pushing the universe apart, or of the Big Bang itself. It would open vast realms of time and space and energy to science and speculation.
+               <p class="main-article">If corroborated, Dr. Kovac’s work will stand as a landmark in science comparable to the recent discovery of dark energy pushing the universe apart, or of the Big Bang itself. It would open vast realms of time and space and energy to science and speculation.
 
                Confirming inflation would mean that the universe we see, extending 14 billion light-years in space with its hundreds of billions of galaxies, is only an infinitesimal patch in a larger cosmos whose extent, architecture and fate are unknowable. Moreover, beyond our own universe there might be an endless number of other universes bubbling into frothy eternity, like a pot of pasta water boiling over.</p>
```
Comment (christianotieno) [inline newyork.html:123]: ## Add a missing feature
- [ ] Just as pointed out by @Spayco, there will be a need to implement the `Editor's Pick` section on our page as it present on the original page and the project's main objective is to be a clone of the original page. Is this achievable? If so, you might want to implement it using the `aside` tag. I have shared below the comparisons.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/Amalcxc/New-York-Times/one-page-article/newyork.html)|
|:--:|:--:|
|editor's-pick|editor's-pick-clone|

File: newyork.html [hunk 7]
```diff
@@ -98,66 +109,137 @@ <h6>March 17, 2014</h6>
...
 
 
         
-        </div>
+        <div class="theory-container">
+        <!-- the grip photo with words -->
+           <div class="theory-words">
+            <h2>The Theory of Inflation</h2>
+            <p class="h6-theory">Astronomers have found evidence to support the theory of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</p>
+          </div>
```
Comment (christianotieno) [inline newyork.html:146]: ## Possible word-wrap?
- [ ] Would you mind adding a word wrap to this section of the `theory of inflation` to resemble the original?

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/Amalcxc/New-York-Times/one-page-article/newyork.html)|
|:--:|:--:|
|theory-of-inflation-word-wrap|theory-of-inflation-word-wrap-clone|

File: newyork.html [hunk 8]
```diff
@@ -290,7 +374,7 @@ <h3>A Special Time</h3>
                     <div class="news-grid1">
```
Comment (christianotieno) [inline newyork.html:374]: ## Styling  Improvement?
- [ ] Would you mind improving the styling on this section to appear as present on the original page. The elements ought to be compact together in the middle.
- [ ] There is also a section that is missing. The `read 615 comments` and the social links right after the main article. 

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html#commentsContainer)|[Clone Page](https://raw.githack.com/Amalcxc/New-York-Times/one-page-article/newyork.html)|
|:--:|:--:|
|sytle-imporment-suggestion|sytle-imporment-suggestion-clone|

File: newyork.html [hunk 9]
```diff
@@ -522,105 +619,112 @@ <h2>NEWS</h2>
      </div>
 
 
+
      <div class="footer-box">
```
Comment (christianotieno) [inline newyork.html:623]: ## Final Changes in the `footer`
- [ ] The footer might need the `logo` to be added. The font size might also need to be improved. Enlarged and boldened in some instance as shown in the comparison below. 
- [ ] This part also has a sticky part at the bottom that has not been implemented. Is this a possible fix?

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/Amalcxc/New-York-Times/one-page-article/newyork.html)|
|:--:|:--:|
|footer|footer-clone|

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @Amalcxc 👋 

The project is coming along just fine. Keep it up 👏 

I acknowledge the effort you have made to get this right and within the deadline we have. You are doing great. Kindly bear with me as I try to point out some few issues that could make the project get better. 

The explanation given in the Pull Request of this project is valid. I would also suggest that as part of it and having a great Pull request message, you also strive in your next series of changes, hopefully, get to highlight a summary of the work done in this project to act as a guide for the next code reviewers.

In my review, I have highlighted some missing features that have neither been implemented nor could need some improvements here or there. Have a look 👇 
### Happy coding 💻 
#### Christian Otieno (CR)

**Pull request closed without merging**